### PR TITLE
pdfarranger: add libhandy

### LINKED ISSUE #288246: pdfarranger: theme preference greyed out (libhandy missing)
### Describe the bug
In pdfarranger 1.10.1 (from 23.11 stable branch) the option to change the theme in the preferences is greyed out and doesn't respect the globally set dark mode in GNOME. The program offers the hint "(Libhandy missing)" next to the greyed out setting (see image below).

### Steps To Reproduce
1. launch pdfarranger
2. open preferences
3. try changing the theme setting

### Expected behavior
The drop-down menu is clickable and offers several different options to choose from.

### Screenshots
pdfarranger-theme-selection

### Additional context
Manually adding the package libhandy in environment.systemPackages doesn't work.

### Notify maintainers
@symphorien 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
system: "x86_64-linux"
 host os: Linux 6.7.4, NixOS, 23.11 (Tapir), 23.11.20240207.6832d0d
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by chewblacka**
## Description of changes
- add libhandy to buildInputs to close below issue
- remove pname anti-pattern from owner & repo in fetchFromGitHub

Closes https://github.com/NixOS/nixpkgs/issues/288246

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/applications/misc/pdfarranger/default.nix [hunk 1]
```diff
@@ -4,16 +4,17 @@
...
   pname = "pdfarranger";
   version = "1.10.1";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "refs/tags/${version}";
+    owner = "pdfarranger";
+    repo = "pdfarranger";
+    rev = version;
```
Comment (dotlambda) [inline pkgs/applications/misc/pdfarranger/default.nix:17]: I think we should keep
```suggestion
    rev = "refs/tags/${version}";
```

**Review by dotlambda — COMMENTED** [review]
Please use `pyproject = true`.

File: pkgs/applications/misc/pdfarranger/default.nix (hunk 1 above)
Comment (chewblacka) [reply pkgs/applications/misc/pdfarranger/default.nix]: > I think we should keep

Ok I've reverted it back to that.

**Review by dotlambda — APPROVED** [review]
Theme can now be set.

**Pull request merged**